src/test/java/racing/CarTest.java

```diff
@@ -0,0 +1,66 @@
+package racing;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CarTest {
+
+    private static Car car;
+
+    void initTestSetup(String inputValue) {
+        System.setIn(new ByteArrayInputStream(inputValue.getBytes()));
+        car = new Car();
+    }
```

Comment:
```suggestion
    private Car car;

    @BeforeEach
    void initTestSetup() {
        car = new Car();
    }
```

어떤것을 의도하셨는지 잘 모르겠지만, setUp은 위와 같은 형태로 해주시면 될 것 같아요.